core/templates/dev/head/services/AssetsBackendApiService.js

```diff
@@ -216,6 +219,9 @@ oppia.factory('AssetsBackendApiService', [
             ASSET_TYPE_AUDIO)) {
             _fetchFile(explorationId, filename, ASSET_TYPE_AUDIO,
               resolve, reject);
+          } else {
+            _fetchFile(explorationId, filename, ASSET_TYPE_AUDIO,
```

Comment:
is there any difference in `else if` and `else` block? maybe we can just have `else` block? (ditto below)

Reply:
@seanlip, has already covered this in his review!

Reply:
Addressed.